cerberus/cerberus.py

```diff
@@ -298,6 +316,9 @@ def _validate_definition(self, definition, field, value):
             if validator:
                 validator(definition[rule], field, value)
 
+        if 'rename' in definition:
```

Comment:
I've purposely added it last since a rename will change a document field which could break other (custom/user) validators looking for the original field name.